localstack-core/localstack/feature_catalog/service_feature.py

```diff
@@ -0,0 +1,84 @@
+from enum import Enum
+from functools import wraps
+
+
+class SupportStatus(Enum):
+    """
+    Indicates the support on LocalStack in regard to the actual implementation
+    E.g.
+    SUPPORTED means that everything is behaving as on AWS
+    SUPPORTED_MOCKED_ONLY means that the response is the same as on AWS, but there is no actual behavior behind it (e.g. no database created)
+    SUPPORTED_PARTIALLY_EMULATED means that there is some kind of emulation, but there may be parts missing
+    NOT_SUPPORTED means this is not implemented at all on LS
+    """
+
+    SUPPORTED = 1
+    SUPPORTED_MOCKED_ONLY = 2
+    SUPPORTED_PARTIALLY_EMULATED = 3
+    NOT_SUPPORTED = 4
+
+
+class ImplementationStatus(Enum):
+    """
+    Indicates implementation status on LS
+    E.g.
+    FULLY_IMPLEMENTED means that all (important?) operations are implemented
+    PARTIALLY_IMPLEMENTED means some selected operations are implemented
+    EXPERIMENTAL means that there is some implementation, but this feature is still higly experimental
+    """
+
+    FULLY_IMPLEMENTED = 1
+    PARTIALLY_IMPLEMENTED = 2
+    EXPERIMENTAL = 3
+
```

Comment:
This was mainly for testing purpose, I don't think this is the best classification as it seems a bit confusing